addon/lazy-router.js

```diff
@@ -39,9 +39,29 @@ export default Ember.Router.extend({
       return handler;
     };
   },
+  _getContainer: function() {
+    var hasGetOwner = typeof getOwner === "function";
+    var container = hasGetOwner? getOwner(this) : this.container;
```

Comment:
Check if `getOwner(this);` has a property called based or parent. we might need that to get the root owner, that might be why the package1 factory isn't there... just a hint from memory, I might be mixing things.